client/settings/payment-request-section/index.js

```diff
@@ -10,13 +10,18 @@ import {
 	CardDivider,
 	CheckboxControl,
 } from '@wordpress/components';
+import { addQueryArgs } from '@wordpress/url';
 import interpolateComponents from 'interpolate-components';
 
 /**
  * Internal dependencies
  */
 import CardBody from '../card-body';
 
+const customizeAppearanceURL = addQueryArgs( window.location.href, {
+	area: 'express_checkouts',
+} );
```

Comment:
Although I can't think of a situation that would allow someone to abuse the `window.location.href` approach or result in unpredictable behavior due to some other params being present in the URL, this is what I'd do just in case:

```suggestion
const customizeAppearanceURL = addQueryArgs( 'admin.php', {
	page: 'wc-settings',
	tab: 'checkout',
	section: 'stripe',
	area: 'express_checkouts',
} );
```